Documentation/operator.md

```diff
@@ -1,57 +1,87 @@
-main
----
-title: "Operator CLI Flags"
-description: "Lists of possible arguments passed to operator executable."
-date: 2021-06-18T14:12:33-00:00
-draft: false
-images: []
-menu:
-  docs:
-    parent: "operator"
-weight: 1000
-toc: false
----
+```
+# Operator CLI Flags
```

Comment:
this isn't probably how you expected it to render

![image](https://user-images.githubusercontent.com/2587585/146031709-beb36085-a5de-4fa4-b5d4-c42cb2c4de7b.png)

Eventually I think that the page should be included in https://prometheus-operator.dev/ so I'm not sure that we should remove the [front matter](https://gohugo.io/content-management/front-matter/) header.

Reply:
noted and restored